src/charmed_kubeflow_chisme/lightkube/batch/_many.py

```diff
@@ -93,5 +107,15 @@ def delete_many(
                 "delete_many only supports objects of types NamespacedResource or GlobalResource,"
                 f" got {type(obj)}"
             )
-
-        client.delete(res=obj.__class__, name=obj.metadata.name, namespace=namespace)
+        try:
+            logger.debug(f"Deleting {obj.__class__} {obj.metadata.name}...")
+            client.delete(res=obj.__class__, name=obj.metadata.name, namespace=namespace)
+        except ApiError as error:
+            if error.status.code == 404 and ignore_missing:
+                logger.debug(f"{obj.__class__} {obj.metadata.name} not found!")
```

Comment:
Small suggestion.  This probably also breaks the formatting test but I'm doing this from my browser so cant get the formatting quite right
```suggestion
                logger.debug(f"{obj.__class__} {obj.metadata.name} not found!  Ignoring because ignore_missing={ignore_missing}.")
```